class/defaults.yml

```diff
@@ -1,4 +1,133 @@
 parameters:
   mimir:
-    =_metadata: {}
-    namespace: syn-mimir
+    =_metadata:
+      multi_instance: true
+
+    name: ${_instance}
+    namespace:
+      name: syn-mimir
+      create: true
+      metadata: {}
+
+    secrets:
+      mimir-bucket-secret:
+        stringData:
+          S3_ACCESS_KEY_ID: null
+          S3_SECRET_ACCESS_KEY: null
+          S3_ENDPOINT: null
+
+    charts:
+      mimir-distributed: "3.0.0"
+
+    helm_values:
+      global:
+        extraEnvFrom:
+          - secretRef:
+              name: mimir-bucket-secret
+        podAnnotations:
+          # increase if bucket secrets change
+          bucketSecretVersion: "0"
+
+      mimir:
+        structuredConfig:
+          alertmanager_storage:
+            s3:
+              bucket_name: ${mimir:name}-alertmanager-bucket
+              access_key_id: \${S3_ACCESS_KEY_ID}
+              endpoint: \${S3_ENDPOINT}
+              secret_access_key: \${S3_SECRET_ACCESS_KEY}
+          blocks_storage:
+            backend: s3
+            s3:
+              bucket_name: ${mimir:name}-blocks-bucket
+              access_key_id: \${S3_ACCESS_KEY_ID}
+              endpoint: \${S3_ENDPOINT}
+              secret_access_key: \${S3_SECRET_ACCESS_KEY}
+          ruler_storage:
+            s3:
+              bucket_name: ${mimir:name}-ruler-bucket
+              access_key_id: \${S3_ACCESS_KEY_ID}
+              endpoint: \${S3_ENDPOINT}
+              secret_access_key: \${S3_SECRET_ACCESS_KEY}
+
+      alertmanager:
+        persistentVolume:
+          enabled: true
+        replicas: 3
+        statefulSet:
+          enabled: true
+
+      compactor:
+        persistentVolume:
+          size: 50Gi
+
+      distributor:
+        replicas: 3
+
+      ingester:
+        persistentVolume:
+          size: 50Gi
+        replicas: 4
```

Comment:
Is the ingester intentionally configured with 4 replicas by default?

Reply:
https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/ingester/#replication

https://github.com/grafana/mimir/blob/a13959db5d38ff65c2b7ef52c56331d2f4dbc00c/operations/helm/charts/mimir-distributed/small.yaml#L53